### PR TITLE
Small Fixes

### PR DESCRIPTION
**Pull request by r0l1**
Just some small fixes ;)

### CODE REVIEW

File: DataStream.js [hunk 1]
```diff
@@ -19,20 +19,20 @@ DataStream = function(arrayBuffer, byteOffset, endianness) {
     this.buffer = new ArrayBuffer(arrayBuffer || 1);
   }
   this.position = 0;
-  this.endianness = endianness == null ? DataStream.LITTLE_ENDIAN : endianness;
+  this.endianness = endianness === null ? DataStream.LITTLE_ENDIAN : endianness;
```
Comment (kig) [inline DataStream.js:22]: This is using the == comparison for a reason. Missing function args are undefined, which doesn't === null, but does == null.

**Comment by kig** [comment]
Can you check if the unit test thing passes with these changes? You should probably change "=== null" to "=== undefined" if you want to use the === comparator. Perf-wise not expecting much of a win from the === comparisons, but I like that the === is clearer in intent.

**Comment by r0l1** [comment]
Thanks for having a look into it. However I won't continue to work with DataStreams (-> https://github.com/desertbit/byte-buffer). Feel free to merge or reject.

**Comment by kig** [comment]
Ooh, that looks neat

**Pull request closed without merging**